packages/framer-motion-3d/src/render/utils/set-value.ts

```diff
@@ -57,6 +57,10 @@ export function setThreeValue(
     if (setters[key]) {
         setters[key](instance, values[key])
     } else {
+        if (key === "opacity" && !instance.transparent) {
+            instance.transparent = true
+        }
```

Comment:
Is this a driveby fix?

Reply:
No - either the latest version of Three.js or R3F doesn't set `transparent` to `true` by default.